lib/cartopy/io/shapereader.py

```diff
@@ -362,7 +362,7 @@ def default_downloader():
         To find the path template of the NEShpDownloader:
 
             >>> ne_dnldr = NEShpDownloader.default_downloader()
-            >>> print(ne_dnldr.target_path_template)
+            >>> print(ne_dnldr.target_path_template.replace('\\\\', '/'))
```

Comment:
Doctest only on Linux? It seems like it makes the doc part of doctest a little less readable... ❓